README.md

```diff
@@ -126,20 +126,23 @@ The commands will do the following:
 </details>
 <br>
 
-Connect VS Code to your JFrog Platform instance by clicking on the green Connect ![Connect](resources/readme/connect.png) button or the provided button in the JFrog extension tab:
-![Connect](resources/readme/preview/connect.png)
+Once you installed, You can view the sigh in page by clicking on the Frog extension tab:
+![LoginPage](resources/readme/preview/sighInPage.png)
 
-You can choose one of the following methods to provide VS Code with the JFrog Platform connections details.
+
+Enter your JFrog platform connection details. Switching from Access token to Username and password can be done by clicking the `Using Password` button.
+
+**Note**: If you would like to use custom URLs for Artifactory or Xray, click on 'Advanced'
+
+There are other methods of providing your JFrog Platform credentials to VS Code than manually writing them in:
 
 ### Use the JFrog CLI Credentials
 
-If JFrog CLI is installed on your machibe, and is configured with the JFrog Platform's connection details, use those details to connect VS Code to the JFrog Platform.
+When JFrog CLI is installed and configured with JFrog Platform's connection details, the following popup will appear:
```

Comment:
When JFrog CLI is installed and configured with JFrog Platform's connection details, the following popup will appear:
-->
If JFrog CLI is installed on your machine and is configured with your JFrog Platform connection details, the you should see the message popup:

Notice how the message is cut - 
![image](https://github.com/jfrog/jfrog-vscode-extension/assets/7830056/5de362db-863d-43e0-b596-c7f99c6f901d)

Also, the message inside the popup should be modified to be as follows - 

Popup title text:
Automatic Sign In --> Sign In Using JFrog CLI

Preferred popup body text:
It looks like JFrog CLI is installed on this machine, and is configured with the connection details of XXX.
Would you like to use this configuration?